services/basket/src/test/java/org/cryptimeleon/incentive/services/basket/SmokeTest.java

```diff
@@ -0,0 +1,31 @@
+package org.cryptimeleon.incentive.services.basket;
+
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+@Slf4j
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class SmokeTest {
```

Comment:
What is this class for?

Reply:
It is a smoke test, basically turn the machine on and see if it smokes :)
The idea is to have a test that just makes sure the service starts successfully